### PR TITLE
Sle 15 update

### PR DESCRIPTION
**Pull request by simotek**

### CODE REVIEW

**Comment by simotek** [comment]
Let me just chuck it through obs and make sure I didn't make a dumb mistake

File: package/patterns-yast.spec [hunk 1]
```diff
@@ -133,9 +131,6 @@ Suggests:       sane-backends
 Suggests:       hplip
 # optionally, open source, derived from iscan
 Suggests:       iscan-free
-# themeing for hardcore KDE lovers
-Suggests:       yast2-theme-openSUSE-Crystal
-Suggests:       yast2-theme-openSUSE-Oxygen
 Suggests:       ivtv
```
Comment (lslezak) [inline package/patterns-yast.spec]: I think we can drop these `ivtv*` dependencies as well, these were added as suggestions for `yast2-tv` which has been dropped. Also the `alevt`, `kradio`, `motv`, `nxtvepg` and `v4l-tools` above were suggested for `yast2-tv`.

File: package/patterns-yast.spec [hunk 2]
```diff
@@ -90,27 +92,23 @@ Recommends:     yast2-samba-server
...
-Suggests:       yast2-mail-plugins
 Suggests:       yast2-multipath
-Suggests:       yast2-phone-services
 Suggests:       yast2-snapper
 # #381365
 Suggests:       yast2-squid
+# themeing for hardcore KDE lovers
+Suggests:       yast2-theme-openSUSE-Oxygen
 # see extra-packages for reasons
 Suggests:       star
```
Comment (lslezak) [inline package/patterns-yast.spec]: IIRC `star` was suggested for already dropped `yast2-backup`

File: package/patterns-yast.spec [hunk 3]
```diff
@@ -65,17 +65,19 @@ Requires:       yast2-perl-bindings
 Requires:       yast2-pkg-bindings
 Requires:       yast2-security
 Requires:       yast2-services-manager
-Requires:       yast2-storage-ng
+Requires:       yast2-storage
```
Comment (lslezak) [inline package/patterns-yast.spec]: This is a bit tricky, the problem is that for TW it is not available yet, but there is a separate staging project for submitting the yast2-storage-ng related packages to TW ([openSUSE:Factory:Staging:E](https://build.opensuse.org/project/show/openSUSE:Factory:Staging:E)). In SLE15 it is already included.

So for openSUSE:Factory:Staging:E or SLE15 we need `yast2-storage-ng` for normal TW still the old `yast2-storage`.

I'd use `yast2-storage-ng` here and for Factory SR we should add a comment that it must go to the Staging:E project. If this change is urgent and needs to go to TW earlier than Staging:E (which will very likely run for long time to iron all the storage-ng bugs) we need a separate SR to TW.

File: package/patterns-yast.spec [hunk 4]
```diff
@@ -90,27 +92,23 @@ Recommends:     yast2-samba-server
...
-Recommends:     yast2-fonts
-Recommends:     yast2-journal
 Recommends:     yast2-vpn
+# Recommend NTP at least until boo#936378 is fixed and YaST is not trying to configure a service that's not there
+Recommends:     ntp
 Suggests:       yast2-online-update-configuration
-Suggests:       yast2-product-creator
 Suggests:       autoyast2
 Suggests:       autoyast2-installation
 Suggests:       inst-source-utils
```
Comment (lslezak) [inline package/patterns-yast.spec]: The `inst-source-utils` was suggested for `yast2-product-creator` and can be dropped. The same applies for `createrepo`.

**Review by lslezak — CHANGES_REQUESTED** [review]
`yast2-storage-ng` is actually correct; more cleanup suggested

**Review by lslezak — APPROVED** [review]
LGTM, thank you! 👍

**Pull request merged**